app/controllers/teams/users_controller.rb

```diff
@@ -1,5 +1,26 @@
 class Teams::UsersController < AuthenticatedController
-   before_action -> { authorize team, policy_class: TeamUsersPolicy }
+    before_action -> { authorize team, policy_class: TeamUsersPolicy }
+
+    def new
+      @user = User.new
+    end
+  
+    def create
+      add_email = user_params.require(:email).downcase
+      @user = User.where(email: add_email).first || User.new(email: add_email)
```

Comment:
You could try `find_or_initialize_by` here https://apidock.com/rails/v6.1.3.1/ActiveRecord/Relation/find_or_initialize_by